docs/jellyfin/recommendations.md

```diff
@@ -37,6 +37,16 @@ try any of the following:
 
 This kind of configuration can be useful if you intend to have separate libraries for movies and series.
 
+### Library File Dates
```

Comment:
it is alo reflected up the chain to season and show/series added dates, which is why they matter at all. somehow tie this into your text below pleasem